public/index.html

```diff
@@ -10,7 +10,7 @@
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <title>Rancid Tomatillos</title>
   </head>
-  <body>
+  <body style="margin: 0;">
```

Comment:
I needed to remove the margin from the body for styling. It was giving some weird gaps. Feel free to put this elsewhere. I just didn't want to make a style sheet for just the one style.

Reply:
I'm good with that for now!